openmc/bounding_box.py

```diff
@@ -55,3 +57,19 @@ def upper_right(self) -> np.ndarray:
     @property
     def volume(self) -> float:
         return np.abs(np.prod(self[1] - self[0]))
+
+    def width(self, axis: str):
```

Comment:
Personally I would just make this a `@property` that returns a numpy array of size 3, similar to `center`, `lower_left`, and `upper_right`.